src/openloco/localisation/string_ids.h

```diff
@@ -49,6 +49,28 @@ namespace openloco::string_ids
     constexpr string_id month_short_november = 51;
     constexpr string_id month_short_december = 52;
 
+    constexpr string_id tooltip_close_window = 60;
+
+    constexpr string_id about_locomotion_caption = 68;
+    constexpr string_id about_locomotion_69 = 69;
+    constexpr string_id about_locomotion_70 = 70;
+    constexpr string_id about_locomotion_71 = 71;
+    constexpr string_id about_locomotion_72 = 72;
+    constexpr string_id about_locomotion_73 = 73;
+    constexpr string_id about_locomotion_74 = 74;
+    constexpr string_id about_locomotion_75 = 75;
+    constexpr string_id about_locomotion_76 = 76;
+    constexpr string_id about_locomotion_77 = 77;
+
```

Comment:
Keep lines in the same dialog together

Reply:
The point of the whitespace is to visibly group the unused strings together, though... As they're not used any longer, I think it makes more sense to separate them.